tests/mpd/protocol/test_idle.py

```diff
@@ -206,3 +228,36 @@ def test_player_then_playlist_then_idle_playlist(self):
         self.assertNotInResponse('changed: player')
         self.assertOnceInResponse('changed: playlist')
         self.assertOnceInResponse('OK')
+
+    def test_output_then_idle_toggleoutput(self):
+        self.idle_event('output')
+        self.send_request('idle output')
+        self.assertNoEvents()
+        self.assertNoSubscriptions()
+        self.assertOnceInResponse('changed: output')
+        self.assertOnceInResponse('OK')
+
+
+class IdleHandlerNoneMixerTest(protocol.BaseTestCase):
+    def idle_event(self, subsystem):
+        self.session.on_idle(subsystem)
+
+    def assertEqualEvents(self, events):  # noqa: N802
+        self.assertEqual(set(events), self.context.events)
+
+    def assertEqualSubscriptions(self, events):  # noqa: N802
+        self.assertEqual(set(events), self.context.subscriptions)
+
+    def assertNoEvents(self):  # noqa: N802
+        self.assertEqualEvents([])
+
+    def assertNoSubscriptions(self):  # noqa: N802
+        self.assertEqualSubscriptions([])
+
+    def test_output_then_idle_output(self):
+        self.idle_event('output')
+        self.send_request('idle output')
+        self.assertNoEvents()
+        self.assertNoSubscriptions()
+        self.assertOnceInResponse('changed: output')
+        self.assertOnceInResponse('OK')
```

Comment:
As we don't want to be able to mute/unmute (or enable/disable in MPD speak) - I have removed this as there should not be any ``changed: output`` events sent any longer.